src/bin/mkksiso

```diff
@@ -511,6 +511,10 @@ def MakeKickstartISO(input_iso, output_iso, ks="", add_paths=None,
         for p in add_paths:
             cmd.extend(["-map", p, os.path.basename(p)])
 
+        # include updates image which will be automatically loaded when on correct place
+        if updates_image:
+            cmd.extend(["-map", updates_image, "updates/updates.img"])
+
         if CheckBigFiles(add_paths):
```

Comment:
The updates_image also needs to be checked to see if it is gigantic :)

Reply:
Done.